src/config/oidc.config.ts

```diff
@@ -0,0 +1,10 @@
+import { registerAs } from '@nestjs/config/dist/utils/register-as.util';
```

Comment:
this file needs a discussion

Reply:
It needs refactoring for sure, env variables need to be cleaned up and config files with them. (aad.client.config.ts, aad.config.ts, aad.obo.config.ts, aad.ropc.config.ts and oidc.config.ts). Will raise a card on the board for that.

Reply:
it can be imported shorter:
```
import { registerAs } from '@nestjs/config';
```

Reply:
https://app.zenhub.com/workspaces/cherrytwist-5ecb98b262ebd9f4aec4194c/issues/cherrytwist/server/765